cmd/podman/sign.go

```diff
@@ -141,7 +141,8 @@ func signCmd(c *cli.Context) error {
 			return errors.Wrapf(err, "error creating new signature")
 		}
 
-		sigStoreDir = filepath.Join(sigStoreDir, strings.Replace(repos[0][strings.Index(repos[0], "/")+1:len(repos[0])], ":", "=", 1))
+		trimmedDigest := repos[0][strings.Index(repos[0], "/")+1 : len(repos[0])]
```

Comment:
Can we do a strings.TrimPrefix() instead here, possibly?

Reply:
```trimmedDigest := strings.TrimPrefix(repos[0], strings.Split(repos[0], "/")[0])```

can I directly get this string like```docker.io``` from some struct and trim it?

Reply:
Not sure, but the TrimPrefix/Split here is much improved - very clear what's going on now

Reply:
:ok_hand: